README.md

```diff
@@ -742,6 +742,10 @@ created run configuration to your needs, e.g. by adding an argument and options
 For running tests and individual test cases from the IDE, the [kotest plugin](https://plugins.jetbrains.com/plugin/14080-kotest)
 needs to be installed. Afterwards tests can be run via the green "Play" icon from the gutter as described above.
 
+# Want to Help or have Questions?
+
+We welcome all contributions. If you're interested in contributing, please read our [contributing guide](https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md), and to get a quick answers to any of your questions we recommend you [join our Slack community](https://join.slack.com/t/ort-talk/shared_invite/enQtMzk3MDU5Njk0Njc1LThiNmJmMjc5YWUxZTU4OGI5NmY3YTFlZWM5YTliZmY5ODc0MGMyOWIwYmRiZWFmNGMzOWY2NzVhYTI0NTJkNmY).
```

Comment:
Some nits:
- Lines should be hard-wrapped at column 120. (Also to make commenting on individual lines easier.)
- IIRC we agreed to use passive voice, i.e. don't use "we". -> "All contributions are welcome." Similar below.
- I believe we spell out "you're" -> "you are"
- "we recommend" -> "it is recommended for you to join"

Reply:
Fixed most of the above except the 'we recommend' as it also in other places in the README. "it is recommended for you to join" is incorrect grammar should be ""it is recommended you join" but find this impersonal, prefer a friendlier tone of voice.